src/config/constants.py

```diff
@@ -1,4 +1,6 @@
 PARAM_TYPE_SERVER_FILE = 'server_file'
 PARAM_TYPE_MULTISELECT = 'multiselect'
+PARAM_TYPE_LIST = 'list'
+PARAM_TYPE_DEPENDANT_LIST = 'dependant_list'
```

Comment:
I think this is not exactly "param type" because the type is still "list" (according to the example, you added)

Reply:
Formally you are right. But 
1. Other types (such 'text', 'int', etc.) don't have its own constants, so the file needs to refactor
2. I believe the issue should been separated to its own type, so, it's for future use

Reply:
Can we call it "list subtype" or smth like this? :)
I raised this concern only because I was confused while reading the code